configure.php

```diff
@@ -202,7 +200,7 @@ function delete_files( string|array $paths ) {
 }
 
 if (
-	$standalone && file_exists( __DIR__ . '/buddy.yml' ) && confirm( 'Do you need the Buddy CI configuration? (Alley devs only -- if the plugin is open-source it will not be needed)', false )
+	file_exists( __DIR__ . '/buddy.yml' ) && confirm( 'Do you need the Buddy CI configuration? (Alley devs only -- if the package is open-source it will not be needed)', false )
```

Comment:
maybe I'm not understanding, but why would a package need a buddy config. It will never stand on it's own, right?

Reply:
I mean, it could if we introduced a private package.